app/routes/api-routes.ts

```diff
@@ -1,15 +1,75 @@
 import express from 'express';
 
 import { CovidReportRepository } from '../repository/CovidReportRepository';
+import { MunicipalityRepository } from '../repository/MunicipalityRepository';
 import { aggregateCovidReports } from '../util/report-aggregator';
+import { CovidReport } from '../domain/types';
 
 const router = express.Router();
 const reportRepo = new CovidReportRepository();
+const municipalityRepo = new MunicipalityRepository();
 
 router.get('/aggregated', async (req, res) => {
   const reports = await reportRepo.getLatestCovidReports();
   const aggregated = aggregateCovidReports(reports);
-  res.send(aggregated);
+  res.json(aggregated);
+});
+
+const reportToExposedFormat = (report: CovidReport) => ({
```

Comment:
Kan gjerne opprette et interface for det som returneres, og spesifisere returtype

Reply:
Ser ikke helt poenget med det men greit :)